deploy/README.md

```diff
@@ -243,11 +243,11 @@ You can use `inclusion` or `exclusion` configuration options to further filter m
 </filter>
 ```
 
-This:
+This filter will:
 
 * Trims the service metadata from the metric datapoint.
 * Rename* the label/metadata `container_name` to `container`, and `pod_name` to `pod`.
-* Filters out metrics for which the namespace is not `kube-system`.
+* Filter out metrics for which the namespace is not `kube-system`.
```

Comment:
I wasn't sure about this one -- should it instead say
```
* only apply to metrics with the `kube-system` namespace
```
?

Reply:
Good catch. it's `inclusions`